core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -300,36 +377,41 @@ object ConsumerGroupCommand {
     // `consumer` is only needed for `describe`, so we instantiate it lazily
     private var consumer: KafkaConsumer[String, String] = null
 
-    def list() {
-      adminClient.listAllConsumerGroupsFlattened().foreach(x => println(x.groupId))
+    def listGroups(): List[String] = {
+      adminClient.listAllConsumerGroupsFlattened().map { _.groupId }.toList
```

Comment:
nitpick: since it's a simple statement, maybe parenthesis in `map` would be a little nicer than braces?